models/cart-discount/src/cart-discount/cart-discount-draft/presets/sample-data-goodstore/furniture-bogo.ts

```diff
@@ -0,0 +1,58 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+// TODO: Import when available
+// import {
+//   ProductTypeDraft,
+//   TProductTypeDraft,
+// } from '@commercetools-test-data/product-type';
+import {
+  CartDiscountMultiBuyLineItemsTargetDraft,
+  CartDiscountValueRelativeDraft,
+  selectionMode,
+} from '../../../../index';
+import { stackingMode } from '../../../constants';
+import type { TCartDiscountDraftBuilder } from '../../../types';
+import * as CartDiscountDraft from '../../index';
+
+// TODO: Need product type draft to be created
+// const productTypeDraft = ProductTypeDraft.presets.sampleDataGoodStore
+//   .furniture()
+//   .build<TProductTypeDraft>();
```

Comment:
This hasn't been created yet. Added notes here to go back and include it when available